data/common/__init__.py

```diff
@@ -214,3 +215,25 @@ def fix_unit(unit):
             return "t⋅km"
         case _:
             return unit
+
+
+def format_number(value):
+    """Format a number to a float with 6 significant digits."""
+    if value is None:
+        return "None"
```

Comment:
What's the point of those lines? I don't see why you need them as your checking the type before calling it in the function below.

Reply:
And I think you could remove the function entirely, see my other comment.

Reply:
done